radboudumc-template/templates/new/publications.html

```diff
@@ -0,0 +1,123 @@
+            <h2>{{ cur_year }}</h2>
+            {% for pub_type in list_pub_type %}
+                {% set found = { 'flag': false } %}
+                {% for bibkey in bibkeys_html.keys() %}
+                    {% if bibkeys_html[bibkey]['pub_type'] == pub_type and bibkeys_html[bibkey]['year'] == cur_year %}
+                        {% if found['flag'] == false %}
+                            {% if pub_type == '@Article' %}
+                                <h3>{{ 'Papers in international journals' }}</h3>
+                            {% endif%}
+                            {% if pub_type == '@Preprint' %}
+                                <h3>{{ 'Preprints' }}</h3>
+                            {% endif%}
+                            {% if pub_type == '@InProceedings' %}
+                                <h3>{{ 'Papers in conference proceedings' }}</h3>
+                            {% endif%}
+                            {% if pub_type == '@Conference' %}
+                                <h3>{{ 'Abstracts' }}</h3>
+                            {% endif%}
+                            {% if pub_type in ['@PhdThesis', '@MastersThesis'] %}
+                                <h3>{{ 'Theses' }}</h3>
+                            {% endif%}
+                            {% if pub_type in ['@Book', '@Other'] %}
+                                <h3>{{ 'Other publications' }}</h3>
+                            {% endif%}
+                            <ul>
+                            {% if found.update({'flag': true}) %}{% endif %}
+                        {% endif %}
+                        {{ bibkeys_html[bibkey]['html'].replace('{filename}/pages', SITEURL).replace('.md', '/') }}
+                    {% endif %}
+                {% endfor %}
+                {% if found['flag'] == true %}
+                    </ul>
+                {% endif %}
+            {% endfor %}
+        </div>
+    {% endfor %}
+
+    </div>
+
+<script type="text/javascript">
+    var buttons = document.getElementsByName('year-selection');
+    var year_blocks = document.getElementsByClassName('publication-year');
+
+    for(var i = 0; i < buttons.length; i++) {
+        buttons[i].addEventListener('click', function(event) {
+        var year = this.dataset.year; 
+        var yearsplit = year.split(' ')
+        for(var b = 0; b < year_blocks.length; b++) {
+            if (yearsplit[yearsplit.length-1] == 'before'){
+                if(year_blocks[b].dataset.year <= parseInt(yearsplit[0])) {
+                    year_blocks[b].style.display = 'block';
+                } else {
+                    year_blocks[b].style.display = 'none';
+                }
+    
+            } else {
+                if(year_blocks[b].dataset.year == year) {
+                    year_blocks[b].style.display = 'block';
+                } else {
+                    year_blocks[b].style.display = 'none';
+                }
+            }
+      }
+    });
+  }
+  buttons[0].click()
+</script>
```

Comment:
We can use this script also on personal pages (to show per year or not). Maybe we can move this to a block.

Reply:
done